parlai/scripts/detect_offensive_language.py

```diff
@@ -81,6 +81,8 @@ def report(world, stats):
         return log
 
     def classify(text, stats):
+        if not text:
```

Comment:
i wonder if the right answer here is to just make our utils in `utils/safety.py` robust to empty text

Reply:
that's fair, would catch it at the source...